samples/Ark.ReferenceProject/Core/Ark.Reference.Core.Common/Exceptions/BusinessRuleViolationException.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Ark.Reference.Core.Common.Exceptions
+{
+    /// <summary>
+    /// Exception thrown when a business rule is violated.
+    /// This exception is designed to be created only with a specific rule code and message.
+    /// </summary>
+    [SuppressMessage("Design", "CA1032:Implement standard exception constructors", Justification = "BusinessRuleViolationException is meant to be created only with a rule code and message")]
+    public class BusinessRuleViolationException : Exception
```

Comment:
remove this. use src/common/Ark.Tools.Core/BusinessRuleViolation/BusinessRuleViolationException.cs